src/components/Articles.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+ import { useParams } from 'react-router-dom';
+ import { getArticles } from '../utilities /api';
+
+ const Articles = () => {
+ 	const [articles, setArticles] = useState([]);
+ 	const { topic } = useParams();
```

Comment:
You're using params here, but your route(s) in App.js aren't set up to handle params yet.